email_account_validity/_store.py

```diff
@@ -31,6 +35,13 @@ def __init__(self, config: dict, api: ModuleApi):
         self._expiration_ts_max_delta = self._period * 10.0 / 100.0
         self._rand = random.SystemRandom()
 
+        use_long_tokens = config.get("send_links", True)
```

Comment:
I don't think the storage layer should be inspecting the config to decide if we're using a long or short token. Apart from anything else, it has the danger of getting out of sync with the logic layer.

I would give `set_renewal_token_for_user` and friends separate `long_renewal_token` and `short_renewal_token` parameters.